app/display.c

```diff
@@ -83,39 +83,33 @@ void display_init(void)
     set_foreground_colour(BLACK);
     set_background_colour(WHITE);
     clear_screen_region(0, 0, 0, 27);
-    prints(0, 0, "     Memtest86+ v6.00b1");
+    prints(0, 5, "Memtest86+ v6.00b1");
```

Comment:
Indent the string by 5 spaces so that we don't lose the visual cue for where this will appear on the screen.

Reply:
Good idea, done.